infra/README

```diff
@@ -0,0 +1,22 @@
+# Onboarding for development deployments
+
+This is a one time setup that enables you to deploy applications to a development AWS accout for testing.
+This bootstraps the account with your personal VPC.
+
+1. `cd infra/bootstrap`
+2. `export DEV_NAME="<your name>"`
+3. `pulumi login --local`
+4. `pulumi stack select staging-dev`
+5. `pulumi up`
+
+# Creating a new project
+
+1. Vault in as developer admin
+1. `pulumi login "s3://boundless-pulumi-state?region=us-west-2&awssdk=v2"`
+1. `pulumi new aws-typescript --secrets-provider="awskms:///arn:aws:kms:us-west-2:968153779208:alias/pulumi-secrets-key" --name prover --stack staging --yes`
+1. `pulumi stack init --secrets-provider="awskms:///arn:aws:kms:us-west-2:968153779208:alias/pulumi-secrets-key" prod`
+1. `pulumi login --local`
+1. `pulumi stack init dev`
+1. `git add -u && git commit -m "<commit message>"`
+1. Add `Pulumi.dev.yaml` to `.gitignore`
+1. `git add -u && git commit -m ".gitignore"`
```

Comment:
Users that can't access `boundless-pulumi-state` or `pulumi-secrets-key` won't be able to access this. Who is this workflow aimed at?

Reply:
Thats a good point. I've split this into 2 parts- one for creating dev stacks which anyone can do, then a part for creating staging/prod which Ops admin permissions are needed for.